app/src/main/java/com/android/sample/ui/components/PlusButtonToCreate.kt

```diff
@@ -34,12 +34,16 @@ fun PlusButtonToCreate(navigationActions: NavigationActions) {
       verticalAlignment = Alignment.CenterVertically,
   ) {
     IconButton(
-        onClick = { navigationActions.navigateTo(TopLevelDestinations.ADD_ACTIVITY) },
-    ) {
-      Icon(
-          imageVector = TopLevelDestinations.ADD_ACTIVITY.icon,
-          contentDescription = TopLevelDestinations.ADD_ACTIVITY.textId,
-      )
-    }
+        onClick = {
+          if (category == "created") {
+            navigationActions.navigateTo(TopLevelDestinations.ADD_ACTIVITY)
+          } else if (category == "enrolled")
+              navigationActions.navigateTo(TopLevelDestinations.OVERVIEW)
+        }) {
+          Icon(
+              imageVector = TopLevelDestinations.ADD_ACTIVITY.icon,
```

Comment:
Here  , there should be a distinction between the 2 cases , because it is not very clear. Why in all cases the icon will be the one of add activity?